src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.qna.UnAuthorizedException;
+import nextstep.users.domain.NsUser;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+
+    private final List<Answer> values;
```

Comment:
이런 컬렉션 타입의 인스턴스 변수들은 필드 초기화를 해서 NPE의 공포에서 좀 더 벗어날 수 있습니다. 

```suggestion
    private final List<Answer> values = new ArrayList<>();
```